src/types/Tags.ts

```diff
@@ -28,3 +39,14 @@ export type TagParsed = TagResult & {
   next: number | null;
   previous: number | null;
 };
+
+// TODO:
+// ordering=
+// name - Aufsteigend
+// -name - Absteigend
+// matching_algorithm - Aufsteigend
+// -matching_algorithm - Absteigend
+// match - Aufsteigend
+// -match - Absteigend
+// document_count - Aufsteigend
+// -document_count - Absteigend
```

Comment:
Whats with those?

Reply:
The sorting for the response of the tags has yet to be implemented.
It is the same scheme as the documents.